test/functional/page_objects/common_page.js

```diff
@@ -79,23 +79,28 @@ export function CommonPageProvider({ getService, getPageObjects, getPageObject }
 
             if (loginPage && !wantedLoginPage) {
               log.debug(`Found loginPage username = ${config.get('servers.kibana.username')}`);
-              await PageObjects.shield.login(
+              return PageObjects.shield.login(
```

Comment:
Hmm, these changes don't look right. They are probably a result of this PR co-existing with https://github.com/elastic/kibana/pull/11881 at one point and all the rebasing that was happening at the time. I'm going to update this PR with only commits related to it. Stay tuned!